src/js/components/App.js

```diff
@@ -52,6 +53,7 @@ var App = React.createClass({
 
     return (
       <div lang={lang} className={classes.join(' ')}>
+        <SkipLinks/>
```

Comment:
I like the idea of having the SkipLinks inside the App component. This allows the consumer app to be agnostic about skip links if they want. :+1: